news/2024/2024-02-16-osumania-7k-world-cup-2024-semifinals-recap.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: post
+title: "osu!mania 7K World Cup 2024: Semifinals Recap"
+date: 2024-02-16 20:00:00 +0000
+---
+
+Catch up with the latest MWC7K happenings, and join us in preparations for the upcoming Finals weekend!
+
+![](/wiki/Tournaments/MWC/2024_7K/img/mwc7k2024-banner.jpg)
+
+The Asian countries have once again shown complete and utter dominance when it comes to 7K mania, once again not allowing any other region to make it past the Semifinals (unlike last year) — and oh boy do next weeks' match-ups look exciting!
+
+However, before looking forward we shall look back once more at the previous week, and everything brought forward by it!
+
+## Mappool showcase
+
+[**Download the pool here! (119 MB)**](https://osu.ppy.sh/beatmaps/packs/P236)
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=v2059913783&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
+</div>
+
+Semifinals not only turned up the difficulty one or two notches, but it was also the first round where the tournament's first [osu! originals](/wiki/osu!_originals) were showcased! Not only that, but we had [Akiri](https://osu.ppy.sh/beatmaps/artists/401)'s big Featured Artist debut as well.
+
+Feel free to check out both the original tracks and mapsets by clicking the banners below. Alternatively, we'd encourage to give the audio previews a shot!
+
+[![](/wiki/shared/news/2024-02-16-osumania-7k-world-cup-2024-semifinals-recap/Kagaribi.jpg)](https://osu.ppy.sh/beatmapsets/2130665#mania/4481018)
+
+<div class="osu-md__paragraph">
+    <audio controls>
+        <source src="https://assets.ppy.sh/artists/365/Songs/Junk_-_Kagaribi.mp3">
+    </audio>
+</div>
+
+[![](/wiki/shared/news/2024-02-16-osumania-7k-world-cup-2024-semifinals-recap/Symphony-Of-The-Lost-Souls.jpg)](https://osu.ppy.sh/beatmapsets/2130621#mania/4480922)
+
+<div class="osu-md__paragraph">
+    <audio controls>
+        <source src="https://assets.ppy.sh/artists/401/The%20Lost%20Symphonies/Akiri%20-%20Symphony%20Of%20The%20Lost%20Souls.mp3">
+    </audio>
+</div>
+
+[![](/wiki/shared/news/2024-02-16-osumania-7k-world-cup-2024-semifinals-recap/Voyage.jpg)](https://osu.ppy.sh/beatmapsets/2130624#mania/4480927)
+
+<div class="osu-md__paragraph">
+    <audio controls>
+        <source src="https://assets.ppy.sh/artists/392/Songs/Sydosys_-_Voyage.mp3">
+    </audio>
+</div>
+
+## Semifinals results
+
+As the final rounds draw nigh, all the teams are starting to show their true colours as we reach endgame levels of difficulty. As such, those who were able to keep up with the sharp increase in difficulty had an easier time this round. Because of that, some teams that initially did quite well in earlier rounds—despite giving their all — weren't able to make a dent on some of the behemoths they were up against.
```

Comment:
```suggestion
As the final rounds draw nigh, all the teams are starting to show their true colours as we reach endgame levels of difficulty. As such, those who were able to keep up with the increased challenge had an easier time this round. Because of that, some teams that initially did quite well in earlier rounds — despite giving their all — weren't able to make a dent on some of the behemoths they were up against.
```